types/material-ui-phone-number/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for material-ui-phone-number 2.2
+// Project: https://github.com/alexplumb/material-ui-phone-number
+// Definitions by: Bassem (NextGLabs) <https://github.com/nextglabs>
+//                 Gelsoh <https://github.com/gelzoh>
+//                 Láďa Durchánek <https://github.com/durchanek>
+//                 Nifanic <https://github.com/nifanic>
```

Comment:
Why are you adding other handles? Those folks will be target of notifications, etc.
If included, you should probably explicitly mention them in PR description, so they could give ACK,
thx!

Reply:
I thought I should give these people some credits, but I removed them for now 👍🏻